build/dockerfiles/rhel.Dockerfile

```diff
@@ -9,6 +9,9 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
+# https://quay.io/repository/eclipse/kubernetes-image-puller?tab=tags
+FROM quay.io/eclipse/kubernetes-image-puller:e28a7fb as k8s-image-puller
```

Comment:
this won't work downstream. quay != authorized registry for Brew

Also the plan was to use the CRW version of this container, not the upstream one

Reply:
see second commit on this PR for the fix. Also need to update get-sources-jenkins.sh to pull the binary for the correct arch when building in Brew.